packages/skill-lesson/hooks/use-next-lesson.ts

```diff
@@ -14,7 +14,7 @@ export const useNextLesson = (
 
   const {data: nextExercise} = trpcSkillLessons.lessons.getNextLesson.useQuery({
     type: lesson._type,
-    slug: router.query.exercise as string,
+    slug: (router.query.exercise as string) || (router.query.lesson as string),
```

Comment:
future: rename all `[exercise]` routes to `[lesson]` and only use the one name, where `lesson` is the general term to describe the objects at these routes